angularjs-portal-home/src/main/webapp/my-app/layout/partials/welcome.html

```diff
@@ -0,0 +1,8 @@
+<div class="welcome-popup center">
+  <h1>Welcome to the new MyUW. Here are the basics.</h1>
+  <img src="img/myuw-welcome.png">
```

Comment:
What can be done to provide equivalent access to non-sighted users in getting the basics about the new MyUW?  Description tagging this image?  Alternative text description of the basics of MyUW available somewhere else, perhaps  linked from a screen-reader-only link at the top of MyUW?

I'd like MyUW to do something to achieve "equivalent access" accessibility of the welcome content, and I'd like to be able to articulate what that something is when describing how MyUW is attending to accessibility.